### PR TITLE
[SEO] meta tags/h1 updates

### PR DESCRIPTION
**Pull request by teallarson**
- Edits per feedback from Victorious SEO.
- Added a meta title and description field to the category page .mdx files so we can customize them more smoothly in the future.

### CODE REVIEW

File: components/home/integrationsHead.tsx [hunk 1]
```diff
@@ -12,10 +12,10 @@ export default function IntegrationsHead({
   return (
     <>
       <Head>
-        <title>{name} Integrations - Best Data App Integration Tools</title>
+        <title>{name} Data Integration ETL Tools - Import &#38; Upload</title>
```
Comment (andyjih) [inline components/home/integrationsHead.tsx]: I actually have a question out with the SEO agency about this one. They made general recommendations to ~all integrations pages which I already merged, but this Salesforce one was an outlier. Mind if we revert this change for now until they get back to me? Sorry about that!

File: pages/integrations/[type]/[plugin].tsx [hunk 2]
```diff
@@ -196,7 +196,9 @@ function DestionationHeaderArrow({ data }) {
       <Row>
         <Col lg={5} md={12}>
           <br />
-          <h1>{data.name} integration for all your tools & customer data</h1>
+          <h1>
```
Comment (andyjih) [inline pages/integrations/[type]/[plugin].tsx]: Let's hold off on this change as well. Sorry!

**Review by andyjih — CHANGES_REQUESTED** [review]
Category Page changes look great. Let's pause on the Integrations pages change for a minute.

**Review by evantahler — APPROVED** [review]
After @andyjih's changes, fine with me!

File: components/home/integrationsHead.tsx [hunk 3]
```diff
@@ -15,7 +15,7 @@ export default function IntegrationsHead({
         <title>{name} Integrations - Best Data App Integration Tools</title>
         <meta
           name="description"
-          content={`The Grouparoo ${name} integration allows you to sync all of your customer data that lives in various sources into ${name}. ✓ Start a free trial now!`}
+          content={`Grouparoo\'s ${name} integration allows you to sync all of your customer data that lives in various sources into ${name}. ✓ Start a free trial now!`}
```
Comment (andyjih) [inline components/home/integrationsHead.tsx]: @tealjulia The apostrophe shows up as below: 
Screen Shot 2021-07-06 at 4 10 40 PM

Evan and I spent a bit of time looking into it and decided the copy change of "The Grouparoo ${name}" gets the job done without the escaping challenges. Thoughts on reverting this change?

**Review by andyjih — CHANGES_REQUESTED** [review]
Just one small change, but open to discussion.

File: components/home/integrationsHead.tsx (hunk 3 above)
Comment (teallarson) [reply components/home/integrationsHead.tsx]: Yeah no problem

**Review by andyjih — APPROVED** [review]
